docs/_pages/releases.md

```diff
@@ -16,6 +16,8 @@ sidebar:
 * Add support for NUnit 4 - [#2483](https://github.com/fluentassertions/fluentassertions/pull/2483)
 * Added `NotBeIn` to check if a `DateTime` is not in a given `DateTimeKind` - [#2536](https://github.com/fluentassertions/fluentassertions/pull/2536)
 * Added `BeNaN` and `NotBeNaN` for assertions on `float` and `double` - [#2606](https://github.com/fluentassertions/fluentassertions/pull/2606)
+* Added options to `.Monitor(...)` extension - [#1954](https://github.com/fluentassertions/fluentassertions/pull/1954)
```

Comment:
🔧 We can remove this one as it has no functional purpose. It's only there to support the `IgnoreEventAccessorExceptions` and `RecordEventsWithBrokenAccessor` methods